valhalla/jawn/src/managers/IntegrationManager.ts

```diff
@@ -94,4 +94,76 @@ export class IntegrationManager extends BaseManager {
       }
     );
   }
+
+  public async getIntegrationByType(integrationType: string): Promise<
+    Result<
+      {
+        id: string;
+        integration_name: string;
+        settings: Record<string, any>;
+        active: boolean;
+      },
+      string
+    >
+  > {
+    const { data, error } = await supabaseServer.client
+      .from("integrations")
+      .select("id, integration_name, settings, active")
+      .eq("organization_id", this.authParams.organizationId)
+      .eq("integration_name", integrationType)
+      .single();
+
+    if (error) {
+      return err(error.message);
+    }
+
+    return ok(
+      data as {
```

Comment:
We are going to start using the `satisfies` keyword whereever it makes sense. It is slightly more type-safe.
```suggestion
      data satistfies {
```

Reply:
Makes sense. Made the required changes and will keep in mind next time!